smarts/core/mission_planner.py

```diff
@@ -193,6 +200,58 @@ def _edge_ids(self, pose: Pose, lane_id: str = None):
 
         return edge_ids
 
+    def cut_in_waypoints(self, pose: Pose, sim, vehicle):
```

Comment:
As Julian mentioned, the general convention is that sim comes at the beginning of the signature.